### DISCUSSION TITLE
Are you gonna update examples?

### DISCUSSION THREAD

**Discussion by tombohub** [remix-run/remix#4120] [Q&A] [answered]
Hello,
 is there a plan to update examples for render to pipe stream? 

Id like to use chakra , but dont know how to make it work.

Or should I use remix version 1.5.1?

**Comment by sergiodxa** [comment, marked as answer]
You don’t need to use render to pipe stream with latest Remix version, you could use the latest Remix and render to a string, so you shouldn’t need to downgrade to 1.5.1
